.github/workflows/bitbar-prepare-artifacts.yaml

```diff
@@ -46,23 +46,23 @@ jobs:
       - name: List build tools versions
         run: ls /Users/runner/Library/Android/sdk/build-tools/
 
-      # Sign auth-debug-androidTest.apk
-      - name: Sign auth-debug-androidTest.apk
+      # Sign app-debug-androidTest.apk
```

Comment:
We plan to retire the `auth` sample app... BitBar however requires an app in order to execute the e2e tests, so I switched to the `app` sample instead...